File: Java/ESAMI/Esame24Gennaio2020/Python/Tennis.py
from Campo import Campo

class Tennis:

    def __init__(self, codice, campo, larghezza, lunghezza, temperatura, costoOrario, terreno) -> None:
        super().__init__(codice, campo, larghezza, lunghezza, costoOrario)
        self.temperatura = temperatura
        self.terreno = terreno

    def __str__(self) -> str:
        return "tennis" + "\t" + self.campo + "\t" + str(self.codice) + "\t" + str(self.larghezza) + "\t" + str(self.lunghezza) + "\t" + str(self.temperatura) + "\t" + self.terreno + "\t" + str(self.costoOrario)

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame24Gennaio2020/Java/Tennis.java
package Esame24Gennaio2020.Java;

public class Tennis extends Campo {
    
    private float temperaturaMedia;
    private String terrno;
    
    public Tennis(int codice, String campo, float larghezza, float lunghezza, float costoOrario, float temperaturaMedia, String terreno) {
        super(codice, campo, larghezza, lunghezza, costoOrario);
        this.temperaturaMedia = temperaturaMedia;
        this.terrno = terreno;
    }

    @Override
    public String toString() {
        return "tennis" + "\t" + campo + "\t" + codice + "\t" + larghezza + "\t" + lunghezza + "\t" + temperaturaMedia + "\t" + terrno + "\t-\t" + costoOrario;
    }
}